src/index.ts

```diff
@@ -1,3 +1,14 @@
-export default function foo(): string {
-	return 'bar';
-}
+export * from './lib/gateway/Gateway';
+export * from './lib/gateway/GatewayDriver';
+export * from './lib/gateway/GatewayStorage';
+export * from './lib/schema/Schema';
+export * from './lib/schema/SchemaEntry';
+export * from './lib/schema/SchemaFolder';

```

Comment:
Maybe make an `index.ts` in each folder so the root-level index.ts can just `export * from ./lib/schema`, `export * from './settings` etc? Can also help imports in other files, just be wary of circulars.

Reply:
This would slow down load times, as it would cause each file to require until the requested export is found.